tools/cmake/cores/Cortex-A5.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020-2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    list(APPEND common_options
+        "-mthumb-interwork"
+        "-marm"
+        "-mfpu=vfpv3"
+        "-mfloat-abi=softfp"
+        "-mno-unaligned-access"
+        "-mcpu=cortex-a5"
+    )
+elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    list(APPEND common_options
+        "-mfpu=vfpv3"
+        "-mfloat-abi=hard"
+        "-mcpu=cortex-a5"
+    )
+endif()
```

Comment:
@ARMmbed/mbed-os-maintainers If we can get some expert to check the architectural features specific to A5, it'd be nice. But otherwise the flags look fairly generic and okay to me.